src/components/molecules/InputCode/InputCode.style.ts

```diff
@@ -1,12 +1,24 @@
 import { css } from '@emotion/react';
+import color from '@/styles/color';
 
 export const inputCodeContainer = css({
   display: 'flex',
   justifyContent: 'space-between',
-  alignItems: 'center',
   width: '100%',
 });
 
-export const inputCodeBtnStyling = css({
-  padding: '10px 25px',
+export const inputCodeBtnStyling = (sendSuccess: boolean) => {
+  if (sendSuccess) {
+    return css({
+      padding: '10px 25px',
+    });
+  }
+  return css({
+    backgroundColor: color.GY[2],
+    padding: '10px 25px',
+  });
+};

```

Comment:
```suggestion
export const inputCodeBtnStyling = (sendSuccess: boolean) => {
  return css({
    padding: '10px 25px',
    backgroundColor: sendSuccess ? null : color.GY[2],
  });
};
```

이렇게 중복된 스타일을 제거하고 삼항연산자로 배경색을 정하는건 어떨까용